test/units/plugins/lookup/test_password.py

```diff
@@ -390,23 +403,30 @@ def test_content_written(self):
 class BaseTestLookupModule(unittest.TestCase):
     def setUp(self):
         self.fake_loader = DictDataLoader({'/path/to/somewhere': 'sdfsdf'})
-        self.password_lookup = password.LookupModule(loader=self.fake_loader)
-        self.os_path_exists = password.os.path.exists
-        self.os_open = password.os.open
-        password.os.open = lambda path, flag: None
-        self.os_close = password.os.close
-        password.os.close = lambda fd: None
-        self.os_remove = password.os.remove
-        password.os.remove = lambda path: None
-        self.makedirs_safe = password.makedirs_safe
-        password.makedirs_safe = lambda path, mode: None
+        self.password_lookup = lookup_loader.get('password')
+        self.password_lookup._loader = self.fake_loader
+        self.os_path_exists = os.path.exists
+        self.os_open = os.open
+        os.open = lambda path, flag: None
+        self.os_close = os.close
+        os.close = lambda fd: None
+        self.os_mkdir = os.mkdir
+        os.mkdir = lambda fd, mode: None
+        self.os_makedirs = os.makedirs
+        os.makedirs = lambda fd, mode: None
+        self.os_remove = os.remove
+        os.remove = lambda path: None
+        self.makedirs_safe = upath.makedirs_safe
+        upath.makedirs_safe = lambda path, mode: None
```

Comment:
why backport all of the unnecessary changes in this file as far back as 2.13?

Reply:
Just a habit, I normally avoid making changes during backports if i can, to minimize introducing  more unknowns